OWNERS

```diff
@@ -4,11 +4,13 @@ reviewers:
 - dofinn
 - mbarnes
 - yithian
+- dustman9000
 approvers:
 - c-e-brumm
 - cblecker
 - dofinn
 - mbarnes
```

Comment:
Would you mind removing Matt while you're in here?

Reply:
done